supplementary_style_guide/style_guidelines/release-notes.adoc

```diff
@@ -1,10 +1,59 @@
 [[release-notes]]
 = Release notes
 
-[[release-notes-doc-texts]]
-== Release notes doc texts
+A release note explains specific product behavior, capability, or problems relevant to a product version. Release notes for a particular product version are collected in a document that is published when the new version is released.
```

Comment:
```suggestion
A release note explains specific product behaviors, capabilities, or problems relevant to a product version. Release notes for a particular product version are collected in a document that is published when the new version is released.
```

Reply:
I think we should stick with the singular here because one release note typically covers one behavior, one capability, or one problem (with exceptions like rebases which typically summarize more). WDYT?

Either way we definitely should fix the singular/plural mismatch that I produced :/

Reply:
```suggestion
A release note explains specific product behavior, capability, or problem relevant to a product version. Release notes for a particular product version are collected in a document that is published when the new version is released.
```

Reply:
I just care about the mismatch so sticking to singular is fine by me.